src/main/resources/db/migration/common/V1_281__courthouse_region.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO courthouse_region_ae (cth_id, reg_id) VALUES (2,5);
+INSERT INTO courthouse_region_ae (cth_id, reg_id) VALUES (3,6);
+INSERT INTO courthouse_region_ae (cth_id, reg_id) VALUES (1,2);
```

Comment:
I think we should check this region value with  @naivinhta - I would expect Swansea to be South West, so `reg_id=3`?

Reply:
Checking ...

Reply:
Nai updated ticket and the sql scripts have been changed to reflect this.